libs/clients/work-machines/src/lib/workMachines.config.ts

```diff
@@ -0,0 +1,25 @@
+import { defineConfig } from '@island.is/nest/config'
+import { z } from 'zod'
+
+const schema = z.object({
+  xRoadServicePath: z.string(),
+  fetch: z.object({
+    timeout: z.number().int(),
+    scope: z.array(z.string()),
+  }),
+})
+
+export const WorkMachinesClientConfig = defineConfig<z.infer<typeof schema>>({
+  name: 'WorkMachinesClientConfig',
+  schema,
+  load: (env) => ({
+    xRoadServicePath: env.required(
+      'XROAD_WORK_MACHINE_LICENSE_PATH',
+      'IS-DEV/GOV/10013/Vinnueftirlitid-Protected/vinnuvelar-token',
+    ),
+    fetch: {
+      timeout: 10000,
+      scope: ['@ver.is/umsyslavinnuvela'],
```

Comment:
Does this scope already exist in all environments?

Should also remove the timeout unless it's needed. The default timeout is now 20000.

Reply:
Yeah, it does.